x-pack/heartbeat/scenarios/stateloader_test.go

```diff
@@ -16,14 +16,14 @@ import (
 var esIntegTwists = framework.MultiTwist(TwistAddRunFrom, TwistMultiRun(3))
 
 func TestStateContinuity(t *testing.T) {
-	scenarioDB.RunAllWithATwist(t, esIntegTwists, func(t *testing.T, mtr *framework.MonitorTestRun, err error) {
+	scenarioDB.RunOneWithATwist(t, esIntegTwists, func(t *testing.T, mtr *framework.MonitorTestRun, err error) {
 		lastSS := framework.LastState(mtr.Events())
 
-		require.Equal(t, monitorstate.StatusUp, lastSS.State.Status)
+		assert.Equal(t, monitorstate.StatusUp, lastSS.State.Status)
```

Comment:
Asserts here make understanding what failed easier. Otherwise you just get the length being wrong.